### DISCUSSION TITLE
[General] Is there any way to claim the Airdrop for Binance accounts?

### DISCUSSION THREAD

**Discussion by Abelaby** [zkSync-Community-Hub/zksync-developers#856] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

My eligible address for the airdrop is a Binance account, since I can't connect with that address, I was checking out the manual claim docs to somehow claim the zk tokens. But seems like it won't work either, have any of you guys found a way to claim the tokens for Binance?

### Repo Link (Optional)

https://github.com/zksync-association/zknation-data?tab=readme-ov-file#l2-addresses

**Comment by alhenaai** [comment]
I'm sorry, but I'm unable to assist with questions related to airdrops or specific account issues on platforms like Binance. For issues related to claiming tokens on Binance, it's best to reach out directly to Binance support for assistance. They can provide the most accurate information regarding the management and claiming of tokens associated with your account on their platform.

Your feedback helps us serve you better. If this fully addresses your question, please give it a thumbs up. If it doesn't, please give it a thumbs down. This will inform the community and team so we can improve our support.

**Comment by theshelb** [comment]
Hey @Abelaby - Shelby from the ZKsync Association here. Could you possibly share some more details around the issues you are experiencing your Binance account? I see on the claim page that it is possible to connect a Binance wallet at https://claim.zknation.io if you go to "All wallets" at the bottom of the Connect Wallet list and search for Binance. Could you please describe what issues you are experiencing and at what step of the connection process to help us figure out where the issue might be?

<img width="345" alt="Screenshot 2024-12-30 at 12 17 11" src="https://github.com/user-attachments/assets/cc838b48-4b18-4856-8ad4-7b2cd1dd3853" />
<img width="344" alt="Screenshot 2024-12-30 at 12 17 23" src="https://github.com/user-attachments/assets/54372e9f-30e6-4c94-810c-88bd590fd053" />

**Comment by Abelaby** [comment]
Hi @theshelb , thank you for responding, I apologize if the way that I described my issue was a bit confusing. The eligible address of mine is not actually a Binance 'wallet'. Not sure how to call it, its the deposit address for my Binance account that I can use to withdraw into fiat from the CEX.

I'm a security researcher from Code4rena so I was using this address to get crypto payments directly to my binance account and then convert it to fiat, its not a traditional wallet address, the CEX deposit address, one which we don't have the private keys. So i'm unable to connect with that address, hope its clear now.

**Reply by theshelb** [reply]
Thanks for the additional details! Will do some asking around and will get back to you ASAP.

**Reply by Abelaby** [reply]
Thank you Shelby.

**Reply by theshelb** [reply]
Hey @Abelaby - given that you do not own the private key to your Binance account (Binance does), you will not be able to connect to claim. This is possibly something Binance could do for you if my understanding is correct, but you would have to reach out to their support. 

I would try reaching out to them with: 
    - The transaction hash (TxID) of the airdrop if you can find it (should be able to find this on your account history from around June 17th)
    - Details of the airdrop (e.g., token name, amount, and eligible address) (see [docs.zknation.io](https://docs.zknation.io/zk-token/zk-airdrop)) 

Binance support will hopefully be able to confirm if the tokens have been received and whether they will credit them to your account. If you need any other information that you are unable to find please let us know!

**Reply by Abelaby** [reply]
Hi @theshelb , I was not able to find the TxID, here are the details.

address: `0x0d4a46ef0b73d60ccebc4d12b95f5ef96563e279`
amount: `16174`
token: ZK token

My friend already reached out the Binance team regarding this but there was no favorable response.

![zksyncClaimPage](https://github.com/user-attachments/assets/d833ca48-d73f-4534-9e7a-449912966163)

**Reply by theshelb** [reply, marked as answer]
I would still try to reach out the Binance with this information as they are the only ones who can support in this matter unfortunately. 

Trying to manual claim won't work in this case either as you would still need the private key to sign transactions.

**Reply by Abelaby** [reply]
Hi @theshelb , I reached out to the Binance team and unfortunately they are also unable to do anything on this matter. Its fine, thank you Shelby for your support.

**Reply by theshelb** [reply]
Really sorry to hear that! Happy to help though.
